integrated_channels/canvas/admin/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""
+Canvas admin integration for configuring Canvas app to communicate with Canvas systems.
+"""
+
+from django.contrib import admin
+
+from integrated_channels.canvas.models import CanvasEnterpriseCustomerConfiguration
+
+
+@admin.register(CanvasEnterpriseCustomerConfiguration)
+class CanvasEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
+    """
+    Django admin model for CanvasEnterpriseCustomerConfiguration.
+    """
+    list_display = (
+        "enterprise_customer_name",
+        "client_id",
+        "client_secret",
+        "canvas_account_id",
+        "canvas_base_url",
+    )
+
+    readonly_fields = (
+        "enterprise_customer_name",
+        "refresh_token",
+    )
+
+    search_fields = ("enterprise_customer_name",)
```

Comment:
hmm, that last comma needed?

Reply:
It's required for tuples yeah

Reply:
I was going off of other integrated channels were using.

Reply:
tuples!!!! got me again